src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -1238,8 +1238,11 @@ private void OnTimeSeriesChange(TimeSeriesChange change)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private void OnChangeInternal(bool triggeredByReplicationThread)
         {
-            if (triggeredByReplicationThread)
+            if (triggeredByReplicationThread &&
+                (ReplicationType == ReplicationLatestEtagRequest.ReplicationType.Internal ||
```

Comment:
actually, can you check if changing the `IsIncomingReplication` is a nicer way to handle this?

Reply:
done